src/main/java/org/jboss/logmanager/ExtHandler.java

```diff
@@ -429,6 +450,32 @@ public boolean isCallerCalculationRequired() {
         return false;
     }
 
+    @Override
+    protected void reportError(String msg, Exception ex, int code) {
+        super.reportError(msg, ex, code);
+    }
+
+    /**
+     * Report an error using a handler's specific error manager, if any.
+     *
+     * @param handler the handler
+     * @param msg the error message
+     * @param ex the exception
+     * @param code the error code
+     */
+    public static void reportError(Handler handler, String msg, Exception ex, int code) {
```

Comment:
Do we want this to be public? No real strong opinion either way, just curious.

Reply:
This one, yeah it's intended for usage by other handlers and making it non-public would complicate things due to it being static. It doesn't really expose anything sensitive; it's just a useful utility method.